modules/post/windows/manage/payload_inject.rb

```diff
@@ -161,13 +161,13 @@ def arch_check(pay,pid)
   def create_temp_proc(pay)
     windir = client.sys.config.getenv('windir')
     # Select path of executable to run depending the architecture
```

Comment:
this is one of those places to observe the need for a 'where's the system directory' higher-level function :)